test/libsolidity/smtCheckerTests/abi/abi_decode_array.sol

```diff
@@ -7,9 +7,7 @@ contract C {
 
 		(uint[] memory c, uint[] memory d) = abi.decode(b1, (uint[], uint[]));
 		assert(a.length == c.length);
-		assert(a.length == d.length); // should fail
 		assert(b.length == d.length);
-		assert(b.length == c.length); // should fail
```

Comment:
The fact that these two removed assertions should fail already followed from the other three assertions.